server/src/main/resources/schema/wildfly-config_4_0.xsd

```diff
@@ -2337,11 +2337,17 @@
         </xs:attribute>
         <xs:attribute name="security-realm" type="xs:string" use="optional" />
         <xs:attribute name="username" type="xs:string" use="optional" />
-        <xs:attribute name="ignore-unused-configuration" type="xs:boolean" default="false">
+        <xs:attribute name="ignore-unused-configuration" type="xs:boolean" use="optional">
             <xs:annotation>
                 <xs:documentation>
-                    Set to true to instruct the master Host Controller to automatically not forward configuration and
+                    When set to true, this instructs the master Host Controller to not forward configuration and
```

Comment:
That already has a slighty more concise version: 

host.write-remote-domain-controller.ignore-unused-configuration=Set to true to instruct the master Host Controller to automatically not forward configuration and operations for profiles, socket binding groups and server groups which do not affect our servers.